js/index.js

```diff
@@ -0,0 +1,16 @@
+const today = new Date();
+const thisYear = today.getFullYear();
+const footer = document.querySelector('footer');
+const copyright = document.createElement('p');
+
+copyright.innerHTML = `Ron Javellana ${thisYear}`;
+footer.appendChild(copyright)
+
+const skills = ["HTML", "CSS", "Ruby", "Javascript", "SQL", "Linux", "Postman"];
+const skillsList = document.querySelector('#skills').lastElementChild;
```

Comment:
This may cause bugs in future updates if you don't remember that this variable is selecting the last element inside of your selector. I sent an example to you in a slack DM. In the future, just add an ID to your HTML element and select that in your javascript. You can also select the _first_ child element inside a parent element by declaring it after a space in your selector like so: document.querySelector('#skills ul')